### DISCUSSION TITLE
Allow `loop` and `playlist` query params for YouTube embeds

### DISCUSSION THREAD

**Discussion by hansottowirtz** [forem/forem#16875] [Feature Requests]
I'd like to include the `loop` and `playlist` query params for a YouTube video, which allows it to loop. Unfortunately these are stripped out.

e.g.
```
{% embed https://www.youtube.com/embed/nMTWrJPksAI?loop=1&playlist=nMTWrJPksAI %}
```

https://www.youtube.com/embed/nMTWrJPksAI?loop=1&playlist=nMTWrJPksAI

**Comment by michael-tharrington** [comment]
I agree that this would be super helpful to have. There have been lots of times in which I wished I could embed a YouTube playlist. I also think it'd be nice if we could embed a YouTube video at a certain timestamp.

**Comment by andr3wV** [comment]
Agreed. I can help contribute to this in the next week.
